packages/messaging/src/ui/components/ContentCardView/ContentCardView.tsx

```diff
@@ -181,7 +199,8 @@ export const ContentCardView: React.FC<ContentViewProps> = ({
           cardVariant === 'SmallImage'
             ? smallImageStyles.container
             : styles.container,
-          styleOverrides?.container
+          styleOverrides?.container,
+          getUnreadBackgroundColor() && { backgroundColor: getUnreadBackgroundColor() }
```

Comment:
These functions cannot be memoized by react, so Id either memoize them or just handle them inline, which reads much cleaner